src/cascadia/TerminalSettingsModel/CascadiaSettings.cpp

```diff
@@ -144,6 +168,25 @@ winrt::hstring CascadiaSettings::GetSerializationErrorMessage()
     return _deserializationErrorMessage;
 }
 
+// Method Description:
+// - Validates a given colour scheme
+// - A colour scheme is valid if it contains values for _all_ the fields

```

Comment:
nit: colour/color.

I don't really mind either spelling, honestly, but the API is named one thing and the comment says another.